python/Dockerfile

```diff
@@ -21,6 +21,8 @@ RUN sed -i -- 's/\"disable_updater\": false/\"disable_updater\": true/g' /google
 RUN mkdir /.ssh
 ENV PATH /google-cloud-sdk/bin:$PATH
 ADD ./ /root/python
+RUN pip install virtualenv virtualenvwrapper
+RUN bash /root/python/venv/bin/activate
```

Comment:
I'm still confused about this bash command.
when we use this in production are the env variables for the virtual environment going to be set?